### DISCUSSION TITLE
Support All Experimental Next.js Params

### DISCUSSION THREAD

**Discussion by ostersoft** [nrwl/nx#23331] [Feature Requests]
This issues has been partially addressed by: https://github.com/nrwl/nx/pull/20836
However there are 3 more parameters not implemented in case someone wants to run the HTTPS mode providing their own certificates, not the auto-generated ones:

        --experimental-https                    Start the server with HTTPS and generate a self-signed certificate
        --experimental-https-key <path>         Path to a HTTPS key file
        --experimental-https-cert <path>        Path to a HTTPS certificate file
        --experimental-https-ca <path>          Path to a HTTPS certificate authority file

https://github.com/vercel/next.js/blob/23955574ae18a81a572d620f299528244e3a1bd1/packages/next/src/cli/next-dev.ts#L137-L139

**Comment by jaysoo** [comment]
PR: https://github.com/nrwl/nx/pull/23334
